test/evergreen.yml

```diff
@@ -5304,6 +5319,7 @@ buildvariants:
     - name: ".workgen-test"
       batchtime: 1440 # 24 hours
     - name: model-unit-test
+    - name: model-test-failure-workloads
```

Comment:
Do we need to run this task for every commit, or it's fine to run it less frequently (e.g. daily)?

Reply:
Yes, we could do this daily (especially at this point when it's not yet testing too many different cases).